src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,217 @@
+  >
+    <template #header>
+      SUSPECTED COMMITS
+    </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="event-details__content-block details-suspected-commit__content-block"
+        data-ripple
+      >
+        <div class="details-suspected-commit__header-row">
+          <div class="details-suspected-commit__left">
+            <div class="details-suspected-commit__left-title">
+              {{ commit.title }}
+            </div>
+          </div>
+          <div class="details-suspected-commit__right">
+            <div class="details-suspected-commit__right_block">
+              <span class="details-suspected-commit__right_block-commitHash">
+                {{ commit.commitHash.substr(0, 7) }}
+              </span>
+            </div>
+          </div>
+        </div>
+        <div class="details-suspected-commit__author-details">
+          <span>
+            {{ commit.author }}
+          </span>
+          <span class="details-suspected-commit__author-details_relative-time">
+            {{ getRelativeTime(commit.date) }}
+          </span>
+        </div>
+      </div>
+    </template>
+    <template #expandButton>
+      {{
+        isMoreCommitsShown
+          ? "Hide"
+          : `${commits.length - 3} more commits in this release`
+      }}
+    </template>
+  </DetailsBase>
+</template>
+
+<script>
+import DetailsBase from "./DetailsBase";
+
+export default {
+  name: "DetailsSuspectedCommits",
+  components: {
+    DetailsBase
+  },
+  props: {
+    /**
+     * Event commits to show
+     */
+    commits: {
+      type: Array,
+      required: true
+    },
+
+    /**
+     * Error environment language
+     */
+    lang: {
+      type: String,
+      default: undefined
+    }
+  },
+  data() {
+    return {
+      /**
+       * Is block expanded.
+       */
+      isMoreCommitsShown: false
+    };
+  },
+  computed: {
+    /**
+     * Displayed commits items
+     */
+    filteredCommits() {
+      return this.commits.length < 4 || this.isMoreCommitsShown
+        ? this.commits
+        : this.commits.slice(0, 3);
+    }
+  },
+  methods: {
+    /**
+     * Return string represent the relative time string.
+     *
+     * @param {string} date - commit date
+     * @returns {string} relative time from today
+     */
+    getRelativeTime(date) {
+      let currentTime = new Date();
+      let commitTime = new Date(date);
+      let diffInSeconds = Math.abs(currentTime - commitTime) / 1000;
+
+      let numberOfYears = Math.floor(diffInSeconds / (60 * 60 * 24 * 365));
+      if (numberOfYears) {
+        return `committed ${
+          numberOfYears === 1 ? "a" : ""
+        } ${numberOfYears} years ago`;
+      }
+
+      let numberOfMonths = Math.floor(diffInSeconds / (60 * 60 * 24 * 30));
+      if (numberOfMonths) {
+        return `committed ${
+          numberOfMonths === 1 ? "a" : ""
+        } ${numberOfMonths} months ago`;
```

Comment:
If there is only one month, it is also necessary to remove `s` at the end. In other places, too